### ISSUE TITLE
The installation is not as accessible as I had hoped

### ISSUE THREAD

**Issue by Midnight1200** [leoetlino/project-restoration-site#1]
I love the 3ds version but after beating it time and time again normally, I decided to stop for a while. After hearing about this project I would like to take time and install it, however I only have a chromebook available. The issue with that is that chromebooks normally can't open or extract 7zip files, just .zip files. I do want to try it out though since it looks like a lot of effort went into it.

**Comment by leoetlino** [comment]
Chromebooks can run Android or normal Linux applications, and it's easy to extract 7-Zip archives once you have that set up:

* To enable Android apps: https://support.google.com/chromebook/answer/7021273?hl=en
* To enable a Linux environment: https://support.google.com/chromebook/answer/9145439?hl=en

Switching to .zip would bloat the download sizes for a platform that realistically only a very small amount of people use. If you were able to mod your 3DS, I'm sure enabling Android/Linux apps should be very easy for you :)

**Comment by Midnight1200** [comment]
Not necessarily. I did mod my 3ds but only by following the guide step by step. Without instructions or anything, I couldn't really do a whole lot. That said, I did find a 7zip app on the playstore that may work, but idk for sure.
